auctex-latexmk.el

```diff
@@ -85,14 +89,30 @@
 ;;;###autoload
 (defun auctex-latexmk-setup ()
   "Add LatexMk command to TeX-command-list."
-  (setq-default TeX-command-list
-                (cons
-                 '("LatexMk" "latexmk %S%(mode) %t" TeX-run-latexmk nil
-                   (plain-tex-mode latex-mode doctex-mode) :help "Run LatexMk")
-                 TeX-command-list)
-                LaTeX-clean-intermediate-suffixes
-                (append LaTeX-clean-intermediate-suffixes
-                        '("\\.fdb_latexmk" "\\.aux.bak" "\\.fls"))))
+  (add-to-list 'TeX-expand-list
+               '("%(-PDF)"
+                 (lambda ()
+                   (if (and (not TeX-Omega-mode)
+                            (or TeX-PDF-mode TeX-DVI-via-PDFTeX))
+                       "-pdf" ""))))
+  (if auctex-latexmk-inherit-TeX-PDF-mode
```

Comment:
It does not work if we change `auctex-latexmk-inherit-TeX-PDF-mode` after initialization.
Would you move `if` expression to the lambda for "%(-PDF)" in `TeX-expand-list`?